### PR TITLE
Activity stream API

### PR DESCRIPTION
**Pull request by 2e2a**
Provides activity stream resources under `/activity_stream/`

Consist of:
- [x] Create resources for the backend activity stream and expose them in the API
- [ ] Consider only followable resources
- [ ] Filter our private resources
- [ ] Limit number or resources / time period
- [ ] Do not include full content in activity name
- [x] Add config parameter

We can merge this pull request at this state and will address the remaining tasks separately.

### CODE REVIEW

**Comment by xi** [comment]
I had issues with the evolution. Another issue was that new events were created, but `/activity_stream/` did not update.

File: src/adhocracy_core/adhocracy_core/sheets/activity.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    target_isheet = ISheet
+
+
+class ActivitySchema(MappingSchema):
+    """Activity entry."""
+
+    subject = Reference(reftype=SubjectReference)
+    type = SingleLine(validator=OneOf([ActivityType.add.value,
+                                       ActivityType.update.value,
+                                       ActivityType.remove.value,
```
Comment (xi) [inline src/adhocracy_core/adhocracy_core/sheets/activity.py]: I assume `ActivityType` is an enum. Is there no shorter way to express this?

File: etc/development.ini.in [hunk 2]
```diff
@@ -53,6 +53,8 @@ adhocracy.thentos_captcha.backend_url = http://localhost:6542/
 adhocracy.thentos_captcha.frontend_url = http://localhost:6542/
 # URL of the Varnish cache
 #adhocracy.varnish_url = http://127.0.0.1:8088
+# Enable activity stream generation
+adhocracy.activity_stream.enabled = True
```
Comment (xi) [inline etc/development.ini.in]: Should be `true` (lowercase)

File: src/adhocracy_core/adhocracy_core/resources/badge.py [hunk 3]
```diff
@@ -122,7 +122,10 @@ def add_badge_assignments_service(context: IPool, registry: Registry,
     """Add `badge_assignments` service to context."""
     creator = options.get('creator')
     registry.content.create(IBadgeAssignmentsService.__identifier__,
-                            parent=context, registry=registry, creator=creator)
+                            parent=context,
+                            registry=registry,
+                            creator=creator,
+                            autoupdated=True)
```
Comment (xi) [inline src/adhocracy_core/adhocracy_core/resources/badge.py]: How is this related to the PR?

**Comment by 2e2a** [comment]
> I had issues with the evolution. Another issue was that new events were created, but /activity_stream/ did not update.

After merging master I cannot reproduce the evolution exception anymore, this is hopefully fixed.

I did not experience the issue with activity_stream not being updated, will have a closer look on it when continuing working on it in further PRs.
